app/src/main/java/org/jellyfin/androidtv/preference/constant/AudioNightModeBehavior.kt

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.preference.constant
+
+import org.jellyfin.androidtv.R
+import org.jellyfin.androidtv.ui.preference.dsl.EnumDisplayOptions
+
+enum class AudioNightModeBehavior {
```

Comment:
I don't think this enum is used, so can be removed?

Reply:
Fixed, thanks!